articles/GPTs for Business.md

```diff
@@ -0,0 +1,698 @@
+| Break Down Complex Queries: Teach the model to deconstruct complicated requests into simpler sub-queries. | If a complex query is asked, break it down into steps and then perform. Example one: asked about customer satisfaction, first fetch sales data, followed by customer feedback ratings. Example two: for a complex query like forecasting sales trends, first retrieve historical sales data, then current market trends, and synthesize these for a comprehensive response. |
+| Handle Errors and Alternatives: Guide the model on how to respond when data can't be accessed or is unavailable. | When you encounter errors suggest alternative methods or direct the user to human support if the required data cannot be retrieved. |
+| Present Data Clearly: Instruct the model to present data in an easily understandable format. | Avoid technical jargon and present Salesforce data in a clear, concise manner suitable for the user’s expertise level. |
+| Cite Sources: Ensure the model always cites the source of its information. | Always mention that the provided information is retrieved from Salesforce to maintain transparency, where possible cite the report or object used with a link to that data. |
+| Leverage Retrieval for Context: Use the retrieval aspect to fetch relevant information from Salesforce as context for the response. | Instruct the model to retrieve specific data from Salesforce, like recent sales figures or customer interactions, to provide context for its generated responses. |
+
+Based on these best practices, our instructional prompt for the GPT model ends up being:
+```markdown
+As a GPT model specialized in Salesforce data queries, your role is to respond to user requests with precise and relevant Salesforce data. 
+
+Retrieve specific Salesforce data relevant to user queries, using precise and Salesforce-specific terminology. 
+
+When receiving a query, convert it into a specific SOQL query using Salesforce-specific terminology. 
+
+For complex requests, break down the query into simpler sub-queries. 
+
+In situations where you cannot retrieve the required data, suggest alternative methods or guide the user to seek help from human support. 
+
+Present data in a clear and understandable manner, and always cite Salesforce as the source of your information.This structured approach ensures that the GPT model effectively handles Salesforce data queries, maintains security and compliance, and continuously improves its responses based on user feedback.
+```
+Now that you have your instructions set, it's time to test & iterative the GPT using real-world questions. Create a set of questions that employees might typically ask about your sales and customer data and use these to evaluate the effectiveness of your GPT. Here's an example:
+| Query | Expected Response | GPT Response | Rating |
```

Comment:
this table actually looks ok, but for consistency i would also recommend switching away from a table here as well